ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -1994,3 +1994,38 @@ def get_logsize(host, logfile):
     """ get current logsize"""
     logsize = len(host.get_file_contents(logfile))
     return logsize
+
+
+def get_platform(host):
+    result = host.run_command([
+        'python3', '-c',
+        'from ipaplatform.osinfo import OSInfo; print(OSInfo().platform)'
+    ], raiseonerr=False)
+    assert result.returncode == 0
+    return result.stdout_text.strip()
+
+
+def install_packages(host, pkgs):
+    '''
+    For install_packages function list is expected for pkgs argument
+    '''
```

Comment:
It's good practice to use a consistent format for the doc strings inside the file:
```
"""Install packages on a remote host.

:param host: the host where the installation takes place
:param pkgs: packages to install, provided as a list of strings
"""
```